src/core/Form/TextInput/TextInput.baseStyles.tsx

```diff
@@ -76,12 +80,18 @@ export const baseStyles = withSuomifiTheme(
 
     &.fi-text-input--error {
       & .fi-text-input_input {
-        border-color: ${theme.colors.alertBase};
+        border: 2px solid ${theme.colors.alertBase};
+        padding-left: ${math(`${theme.spacing.insetL} - 1`)};
```

Comment:
I assume this is going to be replaced with a fixed value?

Reply:
You assume correctly. Changed.